wordpress-mobile/WordPress-Android
# Fix timezone issue with traffic charts

THIS WAS BASED FROM THE WRONG BRANCH.

Fixes #

This fixes blank page on TRAFFIC tab for some timezones.

This is the following PR of https://github.com/wordpress-mobile/WordPress-Android/pull/20387 since it did not resolve the issue.

> [!WARNING]
> Before Merging
> - [ ] Merge https://github.com/wordpress-mobile/WordPress-FluxC-Android/pull/2967
> - [ ] Update FluxC version to FluxC's trunk

-----

## To Test:



1. Log in to JP.
2. Change the device timezone to an edge, -1, or -2.
3. Open the Stats Traffic tab.
4. Tap "By day" and change granularity to "By week".
5. Verify that the page is not blank.
6. Tap "By week" and change granularity to "By month".
7. Verify that the page loads successfully.

-----

## Regression Notes

1. Potential unintended areas of impact

    - None

2. What I did to test those areas of impact (or what existing automated tests I relied on)

    - N/A

3. What automated tests I added (or what prevented me from doing so)

    - The issue was caused by a logic from 3rd party lib (FluxC). It's not possible to add automated tests in WPAndroid.

-----

## PR Submission Checklist:


-----

## Testing Checklist (strike-out the not-applying and unnecessary ones):
